### PR TITLE
Added form for the blog

### PR DESCRIPTION
**Pull request by Dmambo**
## Features Implemented
 
- [x] Created a method called current_user in your ApplicationController to make current user data available to all controllers.
- [x] Created forms to perform the following functions:

- Creates a Post on behalf of the current_user.
- Create a comment on behalf of the current_user.

- [x] Allow Users to add likes to Posts.

### CODE REVIEW

File: app/views/posts/_form_new_post.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+html: {class: "column form"} do |f| %>
+  <%= f.text_field :title, 
+    placeholder: "Title",
+    maxlength: 250,
+    required: true
+  %>
+  <%= f.label :title, "maximum 250 characters" %>
+  <%= f.text_area :text, placeholder: "text ...", required: true %>
+  <%= f.submit "Create Post" %>
+<% end %>
```
Comment (andersonlebon) [inline app/views/posts/_form_new_post.html.erb:11]: https://user-images.githubusercontent.com/65068771/231975500-582d7140-eadc-4702-a5d9-9bad8a936170.mov

- [ ] Kindly make sure the comment is created as well as liking a post, also clicking on a post or a user should bring you to the corresponding details. refer your self in the screencast above 👆

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Dmambo 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/_form_new_post.html.erb (hunk 1 above)
Comment (Dmambo) [reply app/views/posts/_form_new_post.html.erb:11]: Fixed all the functionality stated in the video above

**Review by jerryowusu — APPROVED** [review]
Hi @Dmambo 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

yeasss


## Highlights
Great work in making the requested changes :+1: 
Comment and post created successfully :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**